### PR TITLE
Integrate api in app

### PR DESCRIPTION
**Pull request by DanielMontesGuerrero**
Add api and app integration

Fixes in api:

>  Include sharedWith in lesion endpoint
>  Limit size of images sent in a single request

#35 #45

### CODE REVIEW

**Comment by DanielMontesGuerrero** [comment]
There are some modified files in backend that I'm not sure if I merged correctly, please review

File: api/src/api/routes/user.ts [hunk 1]
```diff
@@ -63,42 +64,76 @@ userRouter.get('/', (async (req, res, next) => {
...
           .send(`The user ${req.params.idUser} wasn't found `);
       }
+      if (user.lesions === undefined) {
+        user.lesions = [];
+      }
+      // for (const lesion of user.lesions) {
+      //   for await (const photo of lesion.photos) {
+      //     await Photo.setImage(photo);
+      //   }
+      // }
```
Comment (EstebanOlmedo) [inline api/src/api/routes/user.ts]: What do we think about erasing this comment?

File: backend/cnn/ResNet/ResNet.py [hunk 2]
```diff
@@ -315,6 +318,33 @@ def test():
...
+    img = convertToOpenCVFormat(img)
+    proccessed_image = main.process_image(img)
+    transform = transforms.ToTensor()
+    input = transform(proccessed_image)
+    return input
+
+
+def predict(img):
+    if os.path.exists(os.path.join(MODEL_DIR), MODEL)):
+      device = "cuda" if torch.cuda.is_available() else "cpu"
```
Comment (EstebanOlmedo) [inline backend/cnn/ResNet/ResNet.py]: I think here the correct line it's
```python
if os.path.exists(os.path.join(MODEL_DIR, MODEL))
```

**Review by EstebanOlmedo — COMMENTED** [review]
I think everything it's correct. There are a few changes, but they're not critical.

**Review by EstebanOlmedo — APPROVED** [review]
I think everything it's right!

**Pull request merged**